packages/core/src/app/ui/icon/IconCardJCB.tsx

```diff
@@ -3,45 +3,20 @@ import React, { FunctionComponent } from 'react';
 import withIconContainer from './withIconContainer';
 
 const IconCardJCB: FunctionComponent = () => (
-    <svg
-        aria-labelledby="iconCardJCBTitle"
-        clipRule="evenodd"
-        fillRule="evenodd"
-        role="img"
-        strokeLinejoin="round"
-        strokeMiterlimit="1.4"
-        viewBox="0 0 480 301"
-        xmlns="http://www.w3.org/2000/svg"
-    >
+    <svg aria-labelledby="iconCardJCBTitle" viewBox="0 0 40 26" xmlns="http://www.w3.org/2000/svg">
         <title id="iconCardJCBTitle">JCB</title>
+        <path d="M0 0h40v26H0z" fill="#fff" />
         <path
-            d="M444 0H24C12 0 0 12 0 24v240.1c0 24 12 36 36 36h408c24 0 36-12 36-36v-228C480 12 468 0 444 0z"
-            fill="#f4f4f4"
```

Comment:
Hey @vitalii-koshovyi just wondering do we need to update all the other icons? Have we tested this for other providers as well?

Reply:
My concern is that we are updating card icons using Adyen's doc, is that a reliable source?

Reply:
@animesh1987  on the screenshot I show the difference between our current icons and updated icons from Adyen cdn. Credit card icons are updated for all providers and will render the same for every provider. Will check manually on other providers before merge.

@bc-peng I believe Adyen has up-to-date icons and I believe we can trust their icon base.
If we have a concern about Adyen as a source of icons, we need to find other sources and update card icons from there, because our credit card icons are a bit outdated.

Reply:
Tagging @valterfatia for visibility.

Reply:
Thanks @animesh1987! Although I do agree with the need to update the credit card marks, I'm concerned about us "brute-forcing" it by adopting Adyen's set. From a design perspective the icons should retain the border-radius and have a consistent height.

From a quick internal investigation I believe @bc-jennlindeman was the product designer who last updated our credit marks as seen on https://www.figma.com/file/f5sfaYcRzWTsK38rOC5Tud/Partner-logos-%26-marks?node-id=669%3A1655&t=aDaOF15PC42jykKk-11 and should be involved in this discussion since she has a wider view of where these marks are used.

Reply:
@valterfatia yeah, seems reasonable. Okay, we can wait for all credit card icons and when they'll be ready, update them. I'll leave this PR that designers can check what icons need to be updated/added.

Reply:
@valterfatia as you can see we already have icons that are out of our guideline. I believe we need to fix this soon, as our  partners recommend us to update these icons already.